PROD-5: Alinear correctamente los textos en la tabla

johancg04 opened the pull request:
- Se alineo el campo de categoria a la derecha, el campo de precio a la derecha y cantidad a la izquierda.

johancg04 committed:
PROD-5: Alinear correctamente los textos en la tabla
 - Se alineo el campo de categoria a la derecha, el campo de precio a la derecha y cantidad a la izquierda.

Alex-CG commented on view.py:
```diff
@@ -109,9 +109,9 @@ def __init__(self):
-        self.tv.heading('Cantidad', text='Cantidad', anchor=tk.W)
+        self.tv.heading('Categoria', text=categoria_texto, anchor=tk.W)
+        self.tv.heading('Precio', text='Precio', anchor=tk.E)
+        self.tv.heading('Cantidad', text='Cantidad', anchor=tk.E)
```
Los titulos de la cabecera deberian estar centrados en todos los casos, usar: `anchor=CENTER`

johancg04 committed:
PROD-5: Alinear correctamente los textos en la tabla
 - Se alineo el campo de categoria a la derecha, el campo de precio a la derecha y cantidad a la izquierda.
 - Los titulos fueron centrados.

johancg04 committed:
PROD-5: Alinear correctamente los textos en la tabla
 - Se alineo el campo de categoria a la derecha, el campo de precio a la derecha y cantidad a la izquierda.
